test/src/reduced_std_libs.rs

```diff
@@ -1,4 +1,4 @@
-//! This module contains functions for creating reduced versions of the `std` library.
+// This module contains functions for creating reduced versions of the `std` library.
```

Comment:
This was a module doc-comment actually :smile:

Reply:
Ah... that was needed because I was `include!(...)` the `reduced_std_libs` inside the snapshot tests. We don´t need that anymore.